### PR TITLE
SCOOP-19037 Ginti_web_to_store_current_value_of_the_service

### PR DESCRIPTION
**Pull request by rajesh-parjapati**

### CODE REVIEW

File: src/main/java/com/scoperetail/commons/ginti/entity/Service.java [hunk 1]
```diff
@@ -64,6 +64,9 @@ public class Service {
   @Column(name = "sequence_name")
   private String sequenceName;
 
+  @Column(name = "current_seq_value")
```
Comment (vishwanath-mathapati-scope) [inline src/main/java/com/scoperetail/commons/ginti/entity/Service.java:67]: change to current_val

File: src/main/java/com/scoperetail/commons/ginti/model/ServiceRequest.java [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder
+@ToString
+public class ServiceRequest {
+
+    private String service;
+    private boolean current;
+    private int count;
```
Comment (vishwanath-mathapati-scope) [inline src/main/java/com/scoperetail/commons/ginti/model/ServiceRequest.java:42]: change it to fetchCount

File: src/main/java/com/scoperetail/commons/ginti/model/Response.java [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+import java.util.List;
+
+@Getter
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder
+@ToString
+public class Response {
```
Comment (vishwanath-mathapati-scope) [inline src/main/java/com/scoperetail/commons/ginti/model/Response.java:39]: Lets create another response class instead of using old one.

**Review by vishwanath-mathapati-scope — CHANGES_REQUESTED** [review]
Please work on review comments

File: src/main/java/com/scoperetail/commons/ginti/model/ServiceRequest.java [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+
+@Getter
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder
+@ToString
+public class ServiceRequest {
+
+    private String service;
```
Comment (vishwanath-mathapati-scope) [inline src/main/java/com/scoperetail/commons/ginti/model/ServiceRequest.java:40]: Add validation rules here

**Comment by RakshantScopeRetail** [comment]
@vishwanath-mathapati-scope @rajesh-parjapati Please use branch JDK-11 as main branch for cosmos, master has been upgraded to JDK-17

**Comment by vishwanath-mathapati-scope** [comment]
> @vishwanath-mathapati-scope @rajesh-parjapati Please use branch JDK-11 as main branch for cosmos, master has been upgraded to JDK-17

Noted. @RakshantScopeRetail Thanks.
@rajesh-parjapati make a note of this

File: pom.xml [hunk 5]
```diff
@@ -9,15 +9,15 @@
     </parent>
     <groupId>com.scoperetail.commons</groupId>
     <artifactId>ginti-core</artifactId>
-    <version>0.1</version>
+    <version>0.2-SNAPSHOT</version>
```
Comment (RakshantScopeRetail) [inline pom.xml:12]: change version to 0.1.1-SNAPSHOT

**Review by RakshantScopeRetail — CHANGES_REQUESTED** [review]
Request changes

**Pull request closed without merging**